org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/rename/LSPRenameProcessor.java

```diff
@@ -118,8 +118,8 @@ public RefactoringStatus checkInitialConditions(IProgressMonitor pm)
 				}
 			}
 		} catch (Exception e) {
-			handleError(e, status);
-			return new RefactoringStatus();
+			status = new RefactoringStatus();

```

Comment:
Why instantiating a new status here?

Reply:
I also think it is weird, but the previous version of the code was doing it, since I do not understand why it was so, I decided to keep it that way. Since you also do not understand it, I submit a new version of the PR without the instantiation.